website/src/CONTRIBUTING.md

```diff
@@ -0,0 +1,309 @@
+
+
+For example,
+
+![browser preview](/public/Images/guidelines/image30.png)
+
+Let’s say, we want to change the `ndm` to `Node disk manager`, change the title in the corresponding markdown file which is `ndm.md`.
+
+![ndm md file](/public/Images/guidelines/image16.png)
+
+### Change document position
+
+Change the position of the document sidebar item of a particular category by changing the sidebar_position value. Make sure that sibling docs don’t have the same value. As example
+
+![document example](/public/Images/guidelines/image16.png)
+
+
+If you want to change the sidebar position of this document in a concepts category, change the value sidebar_position.
+
+*Change a category name in the docs*
+
+In order to change the category name, update the label value in _category_.json. As example
+
+![change name](/public/Images/guidelines/image6.png)
+
+*Change position of category*
+
+If you want to change the position of the category, change the value of position but make sure that sibling categories don’t have the same position value.
+
+For e.g.
+
+![concepts label](/public/Images/guidelines/image6.png)
+
+### Create a new document and add it to specific category
+
+Let’s say you want to add doc2.md with title Doc 2 title in category concepts as it is in the above example.
+
+
+Create the doc2.md in the folder of category concepts. On the top of the markdown file, add fragment such as
+
+doc2.md
+
+![doc2 md file](/public/Images/guidelines/image8.png)
+
+If you are adding this file in the `next` (unpublished) version of the docs, you are done.
+
+If you are adding it to a specific version of the docs, you will have to update the corresponding version_sidebar json.
+
+For e.g:
+
+If you are updating in the version-1.1.2-sidebars.json, traverse to the concepts category and add these items in the docs.
+
+![doc type](/public/Images/guidelines/image20.png)
+
+Here,  id is path to md file including its name
+
+Also as you can see that this new docs is added at 5th position of the items array. It will come at position number 5 in the sidebar also (in the browser). If you want to change the position, move the object up in items.
+
+By default, position of the doc object in the items array will take precedence over sidebar_position in the markdown file.
+
+
+
+
+### Create a new category
+
+In order to add the category in the docs, add a folder named with category. Inside that folder, add markdown files as well as _category_.json according to your need.
+
+Content of the category json key should match like
+
+![new category object](/public/Images/guidelines/image6.png)
+
+The value of keys may be different for your case.
+
+If you are adding a category in the next (unpublished) version of the docs, you are good to go now.
+
+But if you are adding this category in some version, then you will have to add the category object in the version sidebar json.
+
+For example, for  v1.1.2, change the content in version-1.1.2-sidebars.json 
```

Comment:
v1.1.2 => hightlight it please

Reply:
version-1.1.2-sidebars.json ==> this one also

Reply:
Please go through the content and highlight the portion of the content which will be helpful to understand the context better.